docs/deploying.md

```diff
@@ -387,7 +531,7 @@ procedure. The directory includes an example `compose` configuration.
 
 2. Get the `docker-compose` binary.
 
-		$ sudo wget https://github.com/docker/compose/releases/download/1.1.0/docker-compose-`uname  -s`-`uname -m` -O /usr/local/bin/docker-compose
+		$ sudo wget https://github.com/docker/compose/releases/download/1.2.0/docker-compose-`uname  -s`-`uname -m` -O /usr/local/bin/docker-compose
```

Comment:
Depending on when these docs are published, this may need to be tested and changed to docker-compose `1.3`